Owncloud iOs Client/Tabs/FileTab/FileListDBOperations.m

```diff
@@ -164,6 +164,46 @@ + (void) createAFolder: (NSString *)folderName inLocalFolder:(NSString *)localFo
     }
 }
 
+/*
+ * This method receive the new array of the server and store the changes
+ * in the Database and in the tableview
+ * @param requestArray -> NSArray of path items
+ */
++(void)deleteOldDataFromDBBeforeRefresh:(NSArray *) requestArray parent:(FileDto *) parentFileDTO {
+
+    UserDto *activeUser = [ManageUsersDB getActiveUser];
+    NSMutableArray *directoryList = [NSMutableArray arrayWithArray:requestArray];
+
+    NSString *subfolders = [UtilsUrls getServerSubfolders: [ManageUsersDB getActiveUser]];
```

Comment:
Here you are getting the active user again when you got it a few lines behind doing two sql requests